mlir_graphblas/functions.py

```diff
@@ -58,19 +58,21 @@ def get_mlir(self, *, make_private=True):
   indexBitWidth = 64
 }>
 
+#CSX64 = #sparse_tensor.encoding<{
+  dimLevelType = [ "dense", "compressed" ],
+  pointerBitWidth = 64,
+  indexBitWidth = 64
+}>
+
 module  {
-    func private @cast_csr_to_csc(tensor<?x?xf64, #CSR64>) -> tensor<?x?xf64, #CSC64>
+    func private @cast_csr_to_csx(tensor<?x?xf64, #CSR64>) -> tensor<?x?xf64, #CSX64>
+    func private @cast_csc_to_csx(tensor<?x?xf64, #CSC64>) -> tensor<?x?xf64, #CSX64>
+    func private @cast_csx_to_csr(tensor<?x?xf64, #CSX64>) -> tensor<?x?xf64, #CSR64>
+    func private @cast_csx_to_csc(tensor<?x?xf64, #CSX64>) -> tensor<?x?xf64, #CSC64>
     
-    func private @empty(tensor<?x?xf64, #CSR64>, index) -> tensor<?x?xf64, #CSR64>
-    func private @dup_tensor(tensor<?x?xf64, #CSR64>) -> tensor<?x?xf64, #CSR64>
     func private @ptr8_to_tensor(!llvm.ptr<i8>) -> tensor<?x?xf64, #CSR64>
     func private @tensor_to_ptr8(tensor<?x?xf64, #CSR64>) -> !llvm.ptr<i8>
```

Comment:
Ideally, these would be CSX instead of CSR.

Reply:
The changes in https://github.com/metagraph-dev/mlir-graphblas/pull/84/commits/c188c91127553f940b228133029247c2951fa0cd implement this suggestion.